content/doc/book/managing/system-properties.adoc

```diff
@@ -288,7 +288,7 @@ properties:
   since: 1.651
   description: |
     When starting a new run of any AsyncPeriodicWork extension, if this value is non-negative and the existing log file is larger than the specified number of bytes then the log file will be rotated.
-    For fine-grained control of a specific extension you can use the _FullyQualifiedClassName_.logRotateSize system property to only affect a specific extension.
+    For fine-grained control of a specific extension you can use the _FullyQualifiedClassName_.logRotateSize system property to only affect a specific extension.
```

Comment:
What's happening with these? Do we use weird spaces like nbsp?

Reply:
I was just investigating that. I couldn't immediately see what's different about those 2 spaces in my editor or on GitHub. But if they are nbsp I'll fix it. Sorry about that.